ci-operator/step-registry/ipi/deprovision/vsphere/diags/ipi-deprovision-vsphere-diags-ref.yaml

```diff
@@ -1,5 +1,8 @@
 ref:
   as: ipi-deprovision-vsphere-diags
+  best_effort: true
```

Comment:
This should not be best effort, this should always run.

Reply:
Described in a long way in https://github.com/openshift/release/pull/42192#issuecomment-1673930136, but TLDR: it's not changing when the job runs; it adds an opt-in ability for workflow to resign from this job